### ISSUE TITLE
Mountainduck :: webdav 0-size issue (file content erased)

### ISSUE THREAD

**Issue by vbeskrovny** [iterate-ch/cyberduck#12910]
Hello team,

Protocol: webdav
Operation mode: Online
Bug: file content gets erased
Steps to reproduce:

1. copy file
2. rename
3. The file will be 0-size

**Comment by AliveDevil** [comment]
What's the server software you are using? I don't see this here with a WebDAV-enabled Synology NAS:
![2022-02-21-09-47-42](https://user-images.githubusercontent.com/734738/154919801-475c0af2-6afe-4da7-8fad-bbe002948f90.gif)

Please add additional information as described in the [documentation](https://docs.cyberduck.io/mountainduck/support/).

**Comment by vbeskrovny** [comment]
My bad.

Client: mountainduck
Server: nextcloud dav

PS: let me check if I have the same behavior with the nginx backed webdav.

**Comment by vbeskrovny** [comment]
Yes. Plain webdav via nginx works as expected. It is a nextcloud protocol implementation issue.

**Comment by vbeskrovny** [comment]
> What's the server software you are using? I don't see this here with a WebDAV-enabled Synology NAS: ![2022-02-21-09-47-42](https://user-images.githubusercontent.com/734738/154919801-475c0af2-6afe-4da7-8fad-bbe002948f90.gif)
> 
> Please add additional information as described in the [documentation](https://docs.cyberduck.io/mountainduck/support/).

By the way. After copying - I renamed the file.

**Comment by AliveDevil** [comment]
Well … I thought Github were smart enough to automatically play Gifs.
I did show how I created and duplicated the files.

**Comment by AliveDevil** [comment]
Updated the comment to include a video instead of that Gif.
